.betterer.results

```diff
@@ -2988,9 +2992,29 @@ exports[`better eslint`] = {
       [0, 0, 0, "Do not use any type assertions.", "7"],
       [0, 0, 0, "Do not use any type assertions.", "8"]
     ],
+    "public/app/features/plugins/sandbox/custom_formatter.ts:5381": [
```

Comment:
All errors in this file are from a third party file we have to import and rather not modify the source code at all.